### ISSUE TITLE
FxServer tries to start sessionmanager even when OneSync is enabled

### ISSUE THREAD

**Issue by brunosollar** [citizenfx/fivem#2876]
### What happened?

Self-explanatory.

### Expected result

FXServer not forcing the start of _sessionmanager_.

### Reproduction steps

1. Setup a server and enable OneSync.
2. The server should try to start sessionmanager.

### Importancy

Slight inconvenience

### Area(s)

FXServer

### Specific version(s)

Any FXServer version

**Comment by outsider31000** [comment]
that is done in your server.cfg , is onesync supposed to force start it even when you dont have `ensure ensure sessionmanager` ?

**Comment by Ehbw** [comment]
> that is done in your server.cfg , is onesync supposed to force start it even when you dont have `ensure ensure sessionmanager` ?

Not in this case. citizen-server-impl will try to start sessionmanager even if the server is using onesync infinity  https://github.com/citizenfx/fivem/blob/76e67bd59382996a816b66eb5e9760bda2560863/code/components/citizen-server-impl/src/GameServer.cpp#L1379-L1386

**Comment by AvarianKnight** [comment]
I don't think that code is actually used by anything, RPC natives are all done in C++
